core/src/main/java/org/apache/iceberg/ReachableFileUtil.java

```diff
@@ -137,11 +139,21 @@ public static List<String> manifestListLocations(Table table, Set<Long> snapshot
    * @return the location of statistics files
    */
   public static List<String> statisticsFilesLocations(Table table) {
-    List<String> statisticsFilesLocations = Lists.newArrayList();
-    for (StatisticsFile statisticsFile : table.statisticsFiles()) {
-      statisticsFilesLocations.add(statisticsFile.path());
-    }
+    return statisticsFilesLocations(table, statisticsFile -> true);
```

Comment:
nit: can use `Predicates.alwaysTrue()`

Reply:
I know. But in the other places of the iceberg production code only `x -> true` syntax is used.
So, I wanted to keep a similar code style.